app/geant-exporter/ImportProcessConverter.cc

```diff
@@ -136,7 +136,7 @@ ImportModelClass to_import_model(const std::string& g4_model_name)
         {"UrbanMsc",            ImportModelClass::urban_msc},
         {"ICRU73QO",            ImportModelClass::icru_73_qo},
         {"WentzelVIUni",        ImportModelClass::wentzel_VI_uni},
-        {"hBrem",               ImportModelClass::h_brem},
+        {"hBrems",              ImportModelClass::h_brem},
```

Comment:
You're not going to believe this, but even though the Geant4 process class `G4hBremsstrahlung` uses the string "hBrems", the Geant4 *model* class `G4hBremsstrahlungModel` uses the string "hBrem" (singular).

Just kidding, of course you believe it.

So I think all of the *string* changes need to be reverted, but I support your suggestion of normalizing *our* usage of it -- changing the `ImportModelClass` to use `::foo_brems` instead of `::foo_brem`.

Reply:
Yes, this is where I'm trying my best to be careful! :crab: